specs/extract_spec.py

```diff
@@ -0,0 +1,37 @@
+import pandas as pd
+from mamba import description, context, it, before
+from expects import expect, equal
+
+from blsqpy.descriptor import Descriptor
+import blsqpy.extract as extract
+from pandas.testing import assert_frame_equal
+
+with description('extract') as self:
+
+    with it('extract data elements from config'):
+        config = Descriptor.load("./specs/config/sample")
+
+        data_elements = extract.to_data_elements(config.activities.pills)
```

Comment:
Should we have a check that we don't have duplicate activities in config ?

Reply:
as they are build like json "Dict" at least the "code" (pills) is unique